v6/emoji/emoji.py

```diff
@@ -37,13 +38,909 @@ class Plugin(RestExtension):
     def set_site(self, site):
         self.site = site
         roles.register_local_role('emoji', emoji_role)
+        if hasattr(site, 'transforms'):
+            site.transforms.append(Emojis)
+        else:
+            site.transforms = [Emojis]
 
 
 def emoji_role(name, rawtext, text, lineno, inliner, options={}, content=[]):
     text = text.lower()
+    LOGGER.warn('The role :emoji:`{0}` is deprecated. Use |{0}| instead'.format(text))
     node = nodes.image(
         uri='http://www.tortue.me/emoji/{0}.png'.format(text),
         alt=text,
         classes=['emoji'],
     )
     return [node], []
+
+
+class Emojis(Transform):
+    """
+    Replace some substitutions if they aren't defined in the document.
+    """
+    # run before the default Substitutions
+    default_priority = 210
+
+    # list from http://www.tortue.me/
+    emojis = set([
```

Comment:
Is there a way to get this list in a file and/or have a way to update it? New emojis happen every once in a while.